<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>653</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>454</width>
    <height>653</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>454</width>
    <height>653</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/icon/notebook62.png</normaloff>:/new/icon/notebook62.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">   MainWindow{
    border-radius: 1px;
    border: 2px solid rgb(89, 153, 48);
}

QPushButton {
    border-radius: 2px;
    background:transparent;
    color: green;
}


QPushButton:hover{
  background-color: qconicalgradient(cx:0.5, cy:0.522909, angle:179.9, stop:0.494318 rgba(134, 198, 233, 255), stop:0.5 rgba(206, 234, 248, 255));
  border-radius:5px;
}

QPushButton:pressed{
  background-color:#CCCCCC;
  border-radius:5px;
  border: 1px solid #5F92B2;  
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>71</y>
      <width>450</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QScrollBar:vertical {
       border: 2px solid grey;
       background: white;
       width: 15px;
       margin: 22px 0 22px 0;
   }
   QScrollBar::handle:vertical {
       background: rgb(229,243,255);
       min-height: 20px;
   }
   QScrollBar::handle:hover:vertical {
       background: rgb(205,232,255);
       min-height: 20px;
   }
   QScrollBar::handle:vertical:pressed {
       background: rgb(200,200,255);
       min-height: 20px;
   }
   
   
   QScrollBar::add-line:vertical {
       border: 2px solid grey;
       background: white;
       height: 20px;
       subcontrol-position: bottom;
       subcontrol-origin: margin;
   }
   
   QScrollBar::add-line:vertical:hover {
       border: 2px solid grey;
       background: rgb(231,231,231);
       height: 20px;
       subcontrol-position: bottom;
       subcontrol-origin: margin;
   }
   
   QScrollBar::add-line:vertical:pressed {
       border: 2px solid grey;
       background: rgb(206,206,206);
       height: 20px;
       subcontrol-position: bottom;
       subcontrol-origin: margin;
   }
   
   
   QScrollBar::sub-line:vertical {
       border: 2px solid grey;
       background: white;
       height: 20px;
       subcontrol-position: top;
       subcontrol-origin: margin;
   }
   
   QScrollBar::sub-line:vertical:hover {
       border: 2px solid grey;
       background: rgb(231,231,231);
       height: 20px;
       subcontrol-position: top;
       subcontrol-origin: margin;
   }
   
   QScrollBar::sub-line:vertical:pressed {
       border: 2px solid grey;
       background: rgb(206,206,206);
       height: 20px;
       subcontrol-position: top;
       subcontrol-origin: margin;
   }
   
   
   
   QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
       border: 2px solid grey;
       width: 3px;
       height: 3px;
       background: white;
   }
      
   QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
       background: none;
   }</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>40</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8.5pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>搜索联系人或电话</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;查找联系人，手机号码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-image: url(:/new/icon/zoom34.png);
}




</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-image: url(:/new/icon/close.png);
}


QPushButton:hover {
  background-color: #FF0000;
  border-radius:5px;
  border: 1px solid #5F92B2;  
}
\QPushButton:pressed{
  background-color:#CCCCCC;
  border-radius:5px;
  border: 1px solid #5F92B2;  
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;最小化&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-image: url(:/new/icon/mini.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8.5pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>通讯录管理系统 by:叶超 qhy</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolMenuButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>22</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;菜单&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon.qrc">:/new/icon/notebook62.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>632</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab:selected {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #caf7ff, stop: 0.4 #ebfcff,
stop: 0.5 #ebfcff, stop: 1.0 #caf7ff);
}
QTabBar{
font: 8.5pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>家人</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>朋友</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>同事</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>同学</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>所有</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>最近生日</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>添加</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>添加联系人</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
